### ISSUE TITLE
USER ISSUE: Umlaut crashes server

### ISSUE THREAD

**Issue by Threat11** [StrangeLoopGames/EcoIssues#7277]
**Version:** 0.7.3.3 beta

**Steps to Reproduce:**
1. Have an elected leader.
2. Run for Office to become the new Leader.
3. Enter your speach on the web page.
4. Watch your server going offfline.

**Expected behavior:**
Running for Office, normally. just as before	

**Actual behavior:**
Server crashing

**Comment by rainfarmer** [comment]
@Threat11 Does this happen consistently? 
Are you using any special characters in your speech?

**Comment by Threat11** [comment]
@rainfarmer It does - for me and the other 3 people. But as you mention it I think it's possible that I've used so called "Umlaut": Ä,Ü,Ö and ß as it is very common in our language (many european languages use these. for example Austria, Switzerland, Germany, Luxembourg and the whole north of Europe).

It is happening consistently. The speech will be deleted after the server crashed and a small rollback (about 3 mins) is happening too. Even though the game is already translated I noticed that the Web Site of the server is still completely in english. I hope that helps. Don't hesitate to contact me again :)
